docs/supported-services/sqs.rst

```diff
@@ -75,13 +80,61 @@ This Handel file shows an SQS service being configured:
           delay_seconds: 2
           max_message_size: 262140
           message_retention_period: 345601
+          receive_message_wait_time_seconds: 3
           visibility_timeout: 40
 
+This Handel file shows an SQS service being configured with a `Dead-Letter Queue <http://http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html>`_:
+
+.. code-block:: yaml
+
+    version: 1
+
+    name: my-sqs-queue
+
+    environments:
+      dev:
+        queue:
+          type: sqs
+          queue_type: fifo
+          content_based_deduplication: true
+          delay_seconds: 2
+          max_message_size: 262140
+          message_retention_period: 345601
+          receive_message_wait_time_seconds: 3
+          visibility_timeout: 40
+          dead_letter_queue:
+            max_receive_count: 5
+            queue_type: fifo
+            content_based_deduplication: true
+            delay_seconds: 2
+            max_message_size: 262140
+            message_retention_period: 345601
+            receive_message_wait_time_seconds: 4
+            visibility_timeout: 40
+
+.. _dead-letter:
+
+DeadLetterQueue
```

Comment:
Let's move this under the "Parameters" section since it's part of the parameters information. You can change the heading type to be under Parameters by using the next heading down, which would be the "~" character instead of the "-" character.